torchtext/experimental/datasets/language_modeling.py

```diff
@@ -90,6 +87,12 @@ def _setup_datasets(dataset_name, tokenizer=get_tokenizer("basic_english"),
         dataset_tar = download_from_url(URLS[dataset_name], root=root)
         extracted_files = [os.path.join(root, d) for d in extract_archive(dataset_tar)]
 
+    if dataset_name == "WMTNewsCrawl":
+        data_select = ('train',)
```

Comment:
Why is there a hardcoded data_select?

Reply:
There is no validation and test set provided. Would you prefer I added it as a default argument?

Reply:
If this is the case you should error out and say it doesn't exist and have the user explicitly pass in this option. There are a few arguments to be made in favor of this:
- the number of expected return values depends on this option.
- the user might not be aware of the fact that there is no validation and no test dataset.

Reply:
Agreed! I will account for that and push changes later today